ansible/roles/products/hand-e/server/desktop-icons/templates/scripts/start-docker-container-nuc.j2

```diff
@@ -0,0 +1,17 @@
+#jinja2: trim_blocks:False
+#!/bin/bash
+NUC_NAME={{ nuc_username }}
+NUC_ADDRESS={{ nuc_address }}
+LOCAL_IP={{ dhcp_ip }}
+
+ssh -X $NUC_NAME@$NUC_ADDRESS CONTAINER={{ container_name }} SR_BRANCH={{ config_branch }} HAND_SERIALN={{ hand_serial }} ETH_PORT={{ ethercat_interface }} HAND_ID={{ hand_id }} HOST_IP=$LOCAL_IP NUC_ADDRESS=$NUC_ADDRESS 'bash -s' <<'ENDSSH'
+bash $(while [[ $(ss | grep $HOST_IP | grep $NUC_ADDRESS | grep ssh | grep ESTAB | wc -l) -gt 0 ]]; do sleep 1; done ; pkill -INT -f roslaunch) &
+docker stop ${CONTAINER}
+sleep 2
+docker start ${CONTAINER}
+echo "Starting the container..."
+sleep 3
+docker exec --user user ${CONTAINER} bash -c "source /home/user/projects/shadow_robot/base_deps/devel/setup.bash;source /home/user/projects/shadow_robot/base/devel/setup.bash;roscd sr_ethercat_hand_config/launch;git checkout sr_rhand.launch;git checkout sr_lhand.launch;git checkout ${SR_BRANCH}"
```

Comment:
we should double check if we have a script running with the nuc container so we wont conflict with sr_config branches

Reply:
git checkout ${SR_BRANCH}" doesn't overwrite local changes or conflict with sr_config branches.